### PR TITLE
task1/styling

### PR DESCRIPTION
**Pull request by katyhonguyen**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

post screen styling

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

IMG_4993

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @insert pl github username here

### CODE REVIEW

File: src/screens/PostScreen.tsx [hunk 1]
```diff
@@ -1,43 +1,69 @@
...
+      <View style={styles.imageContainer}>
+        <Image
+          source={{
+            uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+          }}
+          style={{
+            width: 400,
+            height: 250,
+            borderRadius: 10,
+          }}
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx:38]: Let's avoid using inline styles. Try to declare a separate image CSS class in the styles.ts file.

Also, the 400px width is too wide for my screen. We need our image to scale according to the screen widths. Consider setting image width relative to the screen width, then determining the height using an aspect ratio value.

File: src/screens/styles.ts [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
+  },
+  reactionContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    paddingBottom: 20,
+    marginVertical: 10,
+    borderBottomWidth: 1,
+    borderBottomColor: '#C4C4C4',
+  },
```
Comment (ronniebeggs) [inline src/screens/styles.ts:42]: Let's use the `space-between` property instead of `space-around`. Also, we should try and apply an equal amount of padding/margin when creating vertical/horizontal styles. Meaning, instead of a singular padding bottom attribute, let's use padding horizontal so our content is centered within the `reactionContainer`.

File: src/screens/styles.ts [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+  username: {
+    marginHorizontal: 10,
+    marginLeft: 25,
+    fontWeight: 'bold',
+  },
+  date: {
+    marginLeft: 230,
+    color: '#979797',
+  },
```
Comment (ronniebeggs) [inline src/screens/styles.ts:33]: We should avoid large absolute margins like this as they're not adaptable to different screen sizes. Consider using flex box properties here to spread the date and other content across the width of the screen.

File: src/screens/styles.ts [hunk 4]
```diff
@@ -1,10 +1,54 @@
...
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    padding: 10,
+    marginVertical: 10,
+  },
+  username: {
+    marginHorizontal: 10,
+    marginLeft: 25,
+    fontWeight: 'bold',
+  },
```
Comment (ronniebeggs) [inline src/screens/styles.ts:29]: We also want to avoid applying spacing styles (margins/paddings) directly to text. The divs which surround text should determine text positioning.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looking good so far! Right now, the implementation uses a lot absolute styling properties. We should try and use flex box properties whenever possible instead! Take a look at [flexbox froggy](https://flexboxfroggy.com/) if you need more help with this!
